dev-docs/source/Standards/DocumentationGuideForDevs.rst

```diff
@@ -247,3 +247,8 @@ For multi-configuration generators such as Visual Studio or XCode you will need
 ::
 
    bin\Debug\MantidPlot -xq docs\runsphinx_doctest.py -R Rebin
+
+Building the HTML Development Documentation
+===========================================
+
+The developer documentation is written as .rst files, the html files can be built using visual studio. Open mantid in visual studio, in the solution explorer right click on the dev-docs-html project (inside ``Documentation/``) and select build. This will build all the development documentation into the mantid build folder under ``dev-docs/html/``.
```

Comment:
This page is generally about building and running the doc tests and not just for Visual Studio so we need to cater for OSX and Linux developers too. How about "... the html files can be built using the ``dev-docs-html``" target. In Visual Studio the target can be found under the ``Documentation`` folder.